CMakeLists.txt

```diff
@@ -825,7 +828,7 @@ OPTION(ALSOFT_REQUIRE_DSOUND "Require DirectSound backend" OFF)
 OPTION(ALSOFT_REQUIRE_MMDEVAPI "Require MMDevApi backend" OFF)
 IF(HAVE_WINDOWS_H)
     # Check MMSystem backend
-    CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H -D_WIN32_WINNT=0x0502)
+    CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H)
```

Comment:
This should be
```
set(CMAKE_REQUIRED_DEFINITIONS -D_WIN32_WINNT=0x0502)
CHECK_INCLUDE_FILES("windows.h;mmsystem.h" HAVE_MMSYSTEM_H)
unset(CMAKE_REQUIRED_DEFINITIONS)
```
if you want the same behaviour as before.

Reply:
I can't fix it in the codedokode's branch.
I'll close this PR and open a new one with requested change.